SYCL/ESIMD/lsc/Inputs/lsc_flat_load.hpp

```diff
@@ -0,0 +1,147 @@
+//==------- lsc_flat_load.hpp - DPC++ ESIMD on-device test -----------------==//
```

Comment:
I suggest to replace "flat" with "usm" everywhere, as that is the user-facing SYCL term for this

Reply:
Addressed in https://github.com/intel/llvm-test-suite/pull/817/commits/497f51eb7efd1d3cb3e07acc2a97faebc2a309da